src/engraving/libmscore/segment.h

```diff
@@ -216,9 +216,6 @@ class Segment final : public EngravingItem
     bool hasElements(track_idx_t minTrack, track_idx_t maxTrack) const;
     bool allElementsInvisible() const;
 
-    double dotPosX(staff_idx_t staffIdx) const { return _dotPosX[staffIdx]; }
-    void setDotPosX(staff_idx_t staffIdx, double val) { _dotPosX[staffIdx] = val; }
```

Comment:
It looks like the `_dotPosX` member of `Segment` can be removed now. It seems that its values are never read anymore.

Reply:
Yup, forgot that. Done now!